components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -505,6 +505,26 @@ - (void)testBottomDrawerScrollingEnabled {
   XCTAssertEqual(self.fakeScrollView.scrollEnabled, NO);
 }
 
+- (void)testSetTrackingScrollView {
```

Comment:
This doesn't seem to be a setter/getter test. Could you suggest a name that's more descriptive of the work being done?

Reply:
Done